docs/02-dynamic-routes.ipynb

```diff
@@ -296,9 +382,9 @@
         }
       ],
       "source": [
-        "from semantic_router.utils.function_call import get_schema\n",
+        "from semantic_router.utils.function_call import get_schemas_openai\n",
         "\n",
-        "schema = get_schema(get_time)\n",
+        "schema = get_schemas_openai([get_time])\n",
```

Comment:
We don't want to build different functions for different LLMs, everything should be standardized across all

Reply:
Hmm, this is tricky as OpenAI only accepts a very specific format for the function schemas as part of Function calling. One which differs from the schemas we were previously using.

We could use `get_schema` and then have a secondary function (specific to the OpenAI LLM) which then converts it into the OpenAI compatible form. But that would be adding extra steps.

Reply:
Actually don't worry I just saw your further explanation here: https://github.com/aurelio-labs/semantic-router/pull/265#discussion_r1585778720

Will implement the way you described in that comment.